src/pools/pool.ts

```diff
@@ -1,4 +1,19 @@
+/**
+ * Contract definition for a poolifier pool.
+ *
+ * @template Data Type of data sent to the worker.
+ * @template Response Type of response of execution.
+ */
 export interface IPool<Data = unknown, Response = unknown> {
+  /**
+   * Shut down every current worker in this pool.
```

Comment:
What happens to the tasks that are currently running/pending to run.

Reply:
Yeah ... I think they just get killed, whatever they are doing.
@pioardi if this is the case, we should create an issue feature-request for providing/selecting a strategy/option whether to kill it with :fire: or to shutdown it gracefully and wait until the worker has finished its work

Reply:
There is already an open enhancement for that [#70](https://github.com/pioardi/poolifier/issues/70), but we need to think carefully about this since that could be in conflict with the maxInactiveTime and slow down performances also.
The map "tasks" into our pool implementations could help on that.